pkg/libmxnet.pkg

```diff
@@ -4,9 +4,9 @@ OPTS = dict(
     name = VAR.fullname,
     description = '''\
 Lightweight, portable deep learning library
- This package provides the core library exposing the MXNet C API.
- .
- It has been built with OpenCV and OpenMP support disabled.
+This package provides the core library exposing the MXNet C API.
+.
```

Comment:
See e.g. section 7.4 of https://www.debian.org/doc/manuals/debian-faq/ch-pkg_basics for an example.  (The indentation error in the original description patch was because I didn't realize the one-space indentation didn't have to be manual too.)